mautic/mautic
# Contact email statistics in lead view

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ N ]
| New feature/enhancement? (use the a.x branch)      | [ N ]
| Deprecations?                          | [ N ]
| BC breaks? (use the c.x branch)        | [ N ]
| Automated tests included?              | [ Y ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This PR adds few email channel statistics info in contact view:
- Total sent
- Open rate (OR)
- Click-through rate (CTR)
- Click-to-open rate (CTOR)
![obraz](https://user-images.githubusercontent.com/8580942/185570687-90da4eea-8616-4ff1-89a8-0e9dceee964c.png)





#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Add new contact and send few emails to generate statistics
3. Open contact view page, statistics should be available as a new tab




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11406"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
